packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -229,42 +638,38 @@ class ResidentWebRunner extends ResidentRunner {
     } on VersionSkew {
       // Thrown if an older build daemon is already running.
       throwToolExit(
-        'Another build daemon is already running with an older version.\n'
-        'Try exiting other Flutter processes in this project and try again.'
-      );
+          'Another build daemon is already running with an older version.\n'
+          'Try exiting other Flutter processes in this project and try again.');
```

Comment:
nit: the second "try" makes it read like "try existing other processes, then try existing them again" :)

Reply:
Fixed!